packages/flutter/lib/src/material/divider.dart

```diff
@@ -36,37 +36,49 @@ import 'theme.dart';
 /// ![](https://flutter.github.io/assets-for-api-docs/assets/material/divider.png)
 ///
 /// ```dart
-/// Widget build(BuildContext context) {
-///   return Center(
-///     child: Column(
-///       children: <Widget>[
-///         Expanded(
-///           child: Container(
-///             color: Colors.amber,
-///             child: const Center(
-///               child: Text('Above'),
-///             ),
-///           ),
-///         ),
-///         const Divider(
-///           color: Colors.black,
-///           height: 20,
-///           thickness: 5,
-///           indent: 20,
-///           endIndent: 0,
-///         ),
-///         Expanded(
-///           child: Container(
-///             color: Colors.blue,
-///             child: const Center(
-///               child: Text('Below'),
-///             ),
-///           ),
-///         ),
-///       ],
-///     ),
-///   );
-/// }
+///   Widget build(BuildContext context) {
+///    return Center(
+///      child: Column(
+///        children: <Widget>[
+///          Expanded(
+///            child: Container(
+///              color: Colors.amber,
+///              child: const Center(
+///                child: Text('Above'),
+///              ),
+///            ),
+///          ),
+///          const Divider(
+///            height: 20,
+///            thickness: 5,
+///            indent: 20,
+///            endIndent: 20,
+///          ),
+///          // Subheader example from Material spec.
+///          // https://material.io/components/dividers#types
+///          Container(
+///            padding: EdgeInsets.only(left: 20),
```

Comment:
If we're going to use const consistently in an example, then you'll need it here too. Best to check this code with analyze.

Reply:
done